moped-editor/src/views/projects/projectView/ProjectNotes.js

```diff
@@ -170,6 +174,16 @@ const ProjectNotes = ({
   const [isDeleteConfirmationOpen, setIsDeleteConfirmationOpen] =
     useState(false);
   const [deleteConfirmationId, setDeleteConfirmationId] = useState(null);
+  const [shouldSyncFromECapris, setShouldSyncFromECapris] = useState(
+    projectData.moped_project[0].should_sync_ecapris_statuses
+  );
```

Comment:
Sort of along the lines of Mike's question about refetching, I'm wondering if we need to use a state variable here, or if we can let the live data control the UI state. I suppose that would add a slight delay in UI feedback while the mutation and refetch happen...

Reply:
actually the delay isnt that bad....i throttled my network and theres a delay but maybe its fine? @mddilley i pushed up an update implementing johns suggestion here, what do you think?

Reply:
Left this comment hanging out after my last review - sorry! I think that removing the state is a great call, and I think that the snackbars provide good feedback when there is either a successful update or an error. We also have seen sneaky bugs in the past with de-synced state so that is nice to avoid as well. Thanks!